### PR TITLE
Dewarns access

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About the PR

Fixes all warnings in the Access module.
Makes PDAs store the id card entity instead of the id card component.
~~Also fixes a bunch of component.Owner accesses because I had to ECS BUIs to get all of them.~~ Done elsewhere
~~Requires https://github.com/space-wizards/RobustToolbox/pull/4079~~ Merged

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs [hunk 1]
```diff
@@ -891,9 +893,10 @@ private void GiveAllAccess(EntityUid entity)
 
     private void RevokeAllAccess(EntityUid entity)
     {
-        _accessSystem.TrySetTags(entity, new string[]{});
+        _accessSystem.TrySetTags(entity, Array.Empty<string>());
     }
 
+#pragma warning disable CA1069
```
[the remark this answers is not in the document]
Comment (TemporalOroboros) [reply Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs:899]: I had to disable a lot of "independent asserts should be grouped in an 'Assert.Multiple'" warnings when I went through the unit tests. (The linter didn't recognise that wrapping an assert around a TryGetComp and having it immediately followed by an assert on the resulting component made them interdependent.) In this case the comments claim that they have the same value for a reason and I haven't looked hard enough to dispute that.

File: Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs [hunk 2]
```diff
@@ -799,7 +803,7 @@ private void RefillGasTank(EntityUid tank, Gas gasType, GasTankComponent? tankCo
 
         var mixSize = tankComponent.Air.Volume;
         var newMix = new GasMixture(mixSize);
-        newMix.SetMoles(gasType, (1000.0f * mixSize) / (Atmospherics.R * Atmospherics.T20C)); // Fill the tank to 1000KPA.
+        newMix.SetMoles(gasType, (1000.0f * mixSize) / (Atmospherics.R * Atmospherics.T20C)); // Fill the tank to 1000KPA. // TODO: Sort out linter warnings.
```
[the remark this answers is not in the document]
Comment (TemporalOroboros) [reply Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs:806]: The warning this refers to is specific to vscode. For some reason it lints for "excessive parentheses". It wants me to get rid of the parentheses in the numerator, but that's probably less clear.

[the remark this answers is not in the document]
Comment (TemporalOroboros) [reply Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs:806]: I could probably poke around the .vscode file and disable it everywhere, but that should probably be its own thing.

File: Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (TemporalOroboros) [reply Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs]: InvokeVerb has always used the name so I don't know why these are separate.

File: Content.Shared/Access/Systems/AccessReaderSystem.cs [hunk 3]
```diff
@@ -17,11 +17,17 @@ public sealed class AccessReaderSystem : EntitySystem
...
         [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
 
+        public const string Sawmill = "access";
+        private ISawmill _sawmill = default!;
+
         public override void Initialize()
         {
             base.Initialize();
+            _sawmill = _log.GetSawmill(Sawmill);
+
```
Comment (metalgearsloth) [inline Content.Shared/Access/Systems/AccessReaderSystem.cs:30]: EntitySystems already have these now.

File: Content.Shared/Access/Systems/AccessReaderSystem.cs [hunk 4]
```diff
@@ -61,7 +67,7 @@ private void OnInit(EntityUid uid, AccessReaderComponent reader, ComponentInit a
             {
                 if (!_prototypeManager.HasIndex<AccessLevelPrototype>(level))
                 {
-                    Logger.ErrorS("access", $"Invalid access level: {level}");
+                    _sawmill.Error($"Invalid access level: {level}");
                 }
             }
         }
```
Comment (metalgearsloth) [inline Content.Shared/Access/Systems/AccessReaderSystem.cs:73]: This should just be a test instead of doing it at runtime.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**